apt-requirements.txt

```diff
@@ -3,6 +3,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Keep sorted.
+# When updating this file also check if doc/ug/install_instructions/index.md
+# needs to be updated as well.
```

Comment:
Is it worth pointing out that this list does get included in the install instructions where apt is mentioned?